README.md

```diff
@@ -101,3 +101,57 @@ services:
     volumes:
       - ./consul.json:/config/consul.json
 ```
+
+- RabbitMQ
+```yaml
+version: '3.4'
+
+services:
+  rabbitmq:
+    image: rabbitmq:3.7-management-alpine
+    restart: unless-stopped
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+    environment:
+      - RABBITMQ_DEFAULT_USER=
+      - RABBITMQ_DEFAULT_PASS=
+      - RABBITMQ_DEFAULT_VHOST=/
+      - RABBITMQ_HIPE_COMPILE=0
+    volumes:
+      - rabbitmq:/var/lib/rabbitmq
+    hostname: rabbitmq
+
+volumes:
+  rabbitmq:
+```
+
+- Redis
+```yaml
+version: '3.4'
+
+services:
+  redis:
+    image: redis:5-alpine
+    restart: unless-stopped
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis:/data
+    command: redis-server --appendonly yes
```

Comment:
Hm, do we need this? I thought it should start without any additional commands 🤔

Reply:
It's for data persistence, in case of server restart.

Reply:
Ah, got it.